code/modules/power/supermatter/supermatter.dm

```diff
@@ -748,15 +796,15 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	layer = ABOVE_MOB_LAYER
 	moveable = TRUE
 
-/obj/machinery/power/supermatter_crystal/shard/engine
+	/obj/machinery/power/supermatter_crystal/shard/engine
```

Comment:
```suggestion
/obj/machinery/power/supermatter_crystal/shard/engine
```

Reply:
Strong boomer energy, thanks for the pair of eyes